docker/Dockerfile.test

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:20.04
+MAINTAINER radixdlt <devops@radixdlt.com>
+LABEL Description="Java + Ubuntu 16.04 (OpenJDK)"
+
+ENV DEBIAN_FRONTEND noninteractive
+
+
+
+CMD /bin/bash
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends wget curl iputils-ping dnsutils \
+    unzip software-properties-common ca-certificates-java \
+    docker.io
+#install jdk11
+RUN add-apt-repository -y ppa:openjdk-r/ppa && \
+    apt update && \
+    apt install -yq openjdk-11-jdk
```

Comment:
Is there a headless package we could install?  Perhaps this would reduce the number of XWindows packages we need to pull in.